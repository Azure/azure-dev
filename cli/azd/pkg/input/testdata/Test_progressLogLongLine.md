*header*

`prefix`── title ───────────────────────







`prefix`────────────────────────────────

## Next state

*header*

`prefix`── title ───────────────────────



`prefix`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
`prefix`xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
`prefix`xxxxxxxxxxxxxxxx

`prefix`────────────────────────────────
