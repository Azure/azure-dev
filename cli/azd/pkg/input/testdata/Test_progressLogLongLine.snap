<RL>header

<RL><prefix>── title ───────────────────────

<RL>
<RL>
<RL>
<RL>
<RL>

<prefix>────────────────────────────────<RL><M1U><M1U><RL><M1U><RL><M1U><RL><M1U><RL><M1U><RL><RL>
<RL>
<RL><prefix>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
<RL><prefix>xxxxxxxxxxxxxxxxxxxxxxxxxxxxxxxx
<RL><prefix>xxxxxxxxxxxxxxxx

<prefix>────────────────────────────────
